src/EntityFramework/Core/Mapping/DefaultObjectMappingItemCollection.cs

```diff
@@ -821,7 +821,8 @@ private static void ValidateEnumTypeMapping(EnumType edmEnumType, EnumType objec
             var objectComplexType = (ComplexType)containingClrMember.TypeUsage.EdmType;
 
             // Get the type mapping for the complex type
-            var complexTypeMapping = LoadTypeMapping(edmComplexType, objectComplexType, ocItemCollection, typeMappings);
+            // LAJ LAJ - needed for side-effects?
+            LoadTypeMapping(edmComplexType, objectComplexType, ocItemCollection, typeMappings);
```

Comment:
@ajcvickers - could you confirm for me that I need to keep this line? It wasn't used but I'm not sure whether we need side-effects or not. Feel free to pull someone else in if you think they know this code better.